app/controllers/hostgroups_controller.rb

```diff
@@ -77,9 +76,10 @@ def destroy
 
   def current_parameters
     Taxonomy.as_taxonomy @organization, @location do
-      inherited_parameters = Hostgroup.find(params['hostgroup_parent_id']).inherited_params(true) if params['hostgroup_parent_id']
+      hostgroup_parent_parameters = Hostgroup.authorized(:view_hostgroups).find(params['hostgroup_parent_id']).parameters(true) if params['hostgroup_parent_id']
+      hostgroup_parameters = params['hostgroup_id'] ? Hostgroup.authorized(:view_hostgroups).find(params['hostgroup_id']).group_parameters : []
       render :partial => "common_parameters/inherited_parameters",
-             :locals => { :inherited_parameters => inherited_parameters }
+             :locals  => { :inherited_parameters => hostgroup_parent_parameters, :parameters => hostgroup_parameters }
```

Comment:
@domcleal other comments have been fixed, I added the comment above with the question of inherited parameters in hostgroup.